docs/guide/hinge-joint-with-backlash.md

```diff
@@ -0,0 +1,50 @@
+## HingeJointWithBacklash
+
+%figure
+
+![HingeJointWithBacklash](images/joints/HingeJointWithBacklash/HingeJointWithBacklash.png)
+
+%end
+
+Derived from [HingeJoint](../reference/hingejoint.md).
+
+### HingeJointWithBacklash Model
+
+```
+HingeJointWithBacklash [
+  SFNode       jointParameters   HingeJointParameters {} 
+  SFFloat      backlash          0.1    # (0, inf)
+  SFFloat      gearMass          0.01   # [0, inf)
+  MFNode       device            [ ]    # {RotationalMotor, PositionSensor, Brake, PROTO}
+  MFNode       outputSensor      [ ]    # {PositionSensor}
```

Comment:
It doesn't make sense to have MFNode type here. Why not using SFNode instead?

Reply:
`device` is a MFNode in `HingeJoint` and when attributing `outputSensor` through a IS statement, it throws a parameter mismatch error if `outputSensor` isn't also `MF`. It's an intended behavior according to the [docs](https://cyberbotics.com/doc/reference/proto-definition#is-statements).

> It is also an error if the type of the field being associated does not match the type declared in the PROTO's interface. For example, it is illegal to associate an SFColor with an SFVec3f. It is also illegal to associate a SFColor with a MFColor or vice versa. Results are undefined if a field of a node in the PROTO body is associated with more than one field in the PROTO's interface.

Reply:
OK, I understand. Let's keep it like that. Thank you for the explanation.